projects/ngx-charts-on-fhir/src/lib/data-layer-list/annotation-options/annotation-options.component.spec.ts

```diff
@@ -0,0 +1,76 @@
+  let loader: HarnessLoader;
+
+  beforeEach(async () => {
+    colorService = new DataLayerColorService(palette);
+    await TestBed.configureTestingModule({
+      imports: [ReactiveFormsModule, NoopAnimationsModule, MatInputModule, MatFormFieldModule, ColorPickerModule],
+      declarations: [AnnotationOptionsComponent],
+      providers: [FormBuilder, { provide: DataLayerColorService, useValue: colorService }, { provide: COLOR_PALETTE, useValue: palette }],
+    }).compileComponents();
+
+    fixture = TestBed.createComponent(AnnotationOptionsComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+    loader = TestbedHarnessEnvironment.loader(fixture);
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+
+  it('should update annotation label when input is changed', async () => {
+    let labelInputHarness = await loader.getHarness(MatInputHarness.with({ selector: "[id='annotationLabel']" }));
+    let labelInputControl = fixture.componentInstance.form.controls['label'];
+    const testLabel = 'test label';
+    await labelInputHarness.setValue(testLabel);
+    expect(labelInputControl.value).toBe(testLabel);
+  });
+
+  it('should update annotation yMax when input is changed', async () => {
+    let yMaxInputHarness = await loader.getHarness(MatInputHarness.with({ selector: "[id='yMax']" }));
+    let yMaxInputControl = fixture.componentInstance.form.controls['yMax'];
+    await yMaxInputHarness.setValue('100');
+    expect(yMaxInputControl.value).toBe(100);
+  });
+
+  it('should update annotation yMin when input is changed', async () => {
+    let yMinInputHarness = await loader.getHarness(MatInputHarness.with({ selector: "[id='yMin']" }));
+    let yMinInputControl = fixture.componentInstance.form.controls['yMin'];
+    await yMinInputHarness.setValue('100');
+    expect(yMinInputControl.value).toBe(100);
+  });
+
+  it('should update form when input is change', async () => {
+    let annotation = { label: { content: 'Test' }, yMin: 100, yMax: 200 };
+    component.annotation = annotation;
+    let labelInputControl = fixture.componentInstance.form.controls['label'];
+    let yMaxInputControl = fixture.componentInstance.form.controls['yMax'];
+    let yMinInputControl = fixture.componentInstance.form.controls['yMin'];
+    let labelInputHarness = await loader.getHarness(MatInputHarness.with({ selector: "[id='annotationLabel']" }));
+    let yMinInputHarness = await loader.getHarness(MatInputHarness.with({ selector: "[id='yMin']" }));
+    let yMaxInputHarness = await loader.getHarness(MatInputHarness.with({ selector: "[id='yMax']" }));
+    expect(labelInputControl.value).toBe('Test');
+    expect(yMaxInputControl.value).toBe(200);
+    expect(yMinInputControl.value).toBe(100);
+    expect(await labelInputHarness.getValue()).toBe('Test');
+    expect(await yMinInputHarness.getValue()).toBe('100');
+    expect(await yMaxInputHarness.getValue()).toBe('200');
+  });
```

Comment:
There's no need to check the values of the reactive form controls. 

```suggestion
  it('should update form when input is change', async () => {
    let annotation = { label: { content: 'Test' }, yMin: 100, yMax: 200 };
    component.annotation = annotation;
    let labelInputHarness = await loader.getHarness(MatInputHarness.with({ selector: "[id='annotationLabel']" }));
    let yMinInputHarness = await loader.getHarness(MatInputHarness.with({ selector: "[id='yMin']" }));
    let yMaxInputHarness = await loader.getHarness(MatInputHarness.with({ selector: "[id='yMax']" }));
    expect(await labelInputHarness.getValue()).toBe('Test');
    expect(await yMinInputHarness.getValue()).toBe('100');
    expect(await yMaxInputHarness.getValue()).toBe('200');
  });
```

Reply:
@amolht looks like code is not updated. do you have another commit that didn't get pushed?